src/libs/ReportUtils.ts

```diff
@@ -1169,6 +1169,12 @@ function isJoinRequestInAdminRoom(report: OnyxEntry<Report>): boolean {
     if (!report) {
         return false;
     }
+    if (report.policyID) {
+        const policy = getPolicy(report.policyID);
+        if (!PolicyUtils.isExpensifyTeam(policy.owner) && PolicyUtils.isExpensifyTeam(currentUserPersonalDetails?.login)) {
```

Comment:
@nkdengineer plz add a comment on why we are doing this, can inspire by this comment
https://github.com/Expensify/App/blob/5efea54421f469f574b107ff4ca51f8866478b92/src/pages/workspace/WorkspaceMembersPage.tsx#L328-L331

Reply:
@Pujan92 I updated the comment. Please help to check again.